### PR TITLE
fix: add missed column in response

### PR DESCRIPTION
**Pull request by SwiftXLight**
**Description changes:** 
- Add missed photoUrl column in response for findDoctorsByAvailabilityAndSpeciality service
- Add missed doctor full name in patient appointemnt and patient info with last note
- Add updated swagger examples for availability, appointments and patient with last note

**PR checklist:**
1.1 Common
1.1.1 types for input and output params
1.1.2 no any, should be also added to eslint rules
1.1.3 try/catch for any async function
1.1.4 no magic numbers
1.1.5 compare only with constants not with strings
(instead of user === ‘admin’, better user === roles.admin)
1.1.6 no ternary operator inside ternary operator (bad example: question ? first:
second ? third: fourth)
1.1.7 avoid similar types with duplicating by generics
1.1.8 no console.log in pr
1.1.9 .env file should be in .gitignore
1.1.10 delete commented code if it's not part of documentation
1.1.11 no links in the code, env links should be in env file (for example: server url),
constant links (for example default avatar URL) should be in constant file
1.1.12 constants and function names should be lowercase.
1.1.13 no dates format in the code (like "dd/MM/yyyy”), move it in global constant
variable
1.1.14 import rules. imports should come in a specific order: node modules, absolute
path, relative path
1.1.15 strings should be in the constant variable. Example: instead of ‘15 3 * * *’,
const EACH_DAY_15h_15min = ‘15 3 * * *’

1.2 Backend
1.2.1 swagger for each endpoint
1.2.2 less requests to db, all data should be taken with one query
1.2.3 `public` in method use only is use function externally
1.2.4 use ConfigService instead of `process.env`

New patient appointment response example:
image

Updated swagger examples:
image

### CODE REVIEW

File: src/modules/appointment/entity/appointment.entity.ts [hunk 1]
```diff
@@ -7,30 +9,37 @@ import {
   ManyToOne,
   PrimaryGeneratedColumn,
 } from 'typeorm';
+import PatientWithNotes from '../../patient/dto/patient-with-notes.dto';
```
Comment (lyudad) [inline src/modules/appointment/entity/appointment.entity.ts]: absolute path

File: src/modules/availability/entity/availability.entity.ts [hunk 2]
```diff
@@ -7,17 +9,22 @@ export default class Availability {
   id: number;
 
   @Column()
+  @ApiProperty({ example: 'c3864420-2d55-4483-b861-4970e3b7ea40' })
   uuid: string;
```
Comment (lyudad) [inline src/modules/availability/entity/availability.entity.ts]: do you need both id and uuid?

File: src/modules/appointment/entity/appointment.entity.ts (hunk 1 above)
Comment (SwiftXLight) [reply src/modules/appointment/entity/appointment.entity.ts]: fixed

File: src/modules/availability/entity/availability.entity.ts (hunk 2 above)
Comment (SwiftXLight) [reply src/modules/availability/entity/availability.entity.ts]: deleted id and leaft only uuid

**Review by lyudad — APPROVED** [review]

**Pull request merged**